Client/Shape.h

```diff
@@ -16,12 +16,23 @@ class Shape : public Drawable {
 	/** @brief La liste des sommets. */
 	vector<Vector2D> vertices;
 
+	//VERTICIES DEVRAIT ETRE DANS DRAWABLE/////////////////////////////////////
+
 public:
 	/** @brief Le constructeur par défaut. */
 	Shape();
 
+	/** @brief Le constructeur par Vector2D. */
+	Shape(const Vector2D &);
```

Comment:
A Shape must have at least 2 points (a line) to make sense.
A Shape with only a point is just a ......... point.